ui/app/models/sync/destination.js

```diff
@@ -29,6 +29,13 @@ export default class SyncDestinationModel extends Model {
   })
   secretNameTemplate;
 
+  @attr('object', {
```

Comment:
Could you explain the reasoning for moving this here? The base model should only have attributes that every destination has and as you pointed out, github doesn't support `custom_tags` 🤔

Reply:
full disclosure, the designs had this for a generic form of edit/create on destinations. But you're correct it's not on gh or azure so I'll remove from the base.